rotkehlchen/chain/bitcoin/bch/validation.py

```diff
@@ -0,0 +1,24 @@
+from bip_utils.bech32.bch_bech32 import BchBech32Decoder
+
+from rotkehlchen.chain.bitcoin.bch.constants import CASHADDR_PREFIX
+
+
+def is_valid_bitcoin_cash_address(address: str) -> bool:
+    """Validates a Bitcoin Cash's CashAddr format
+    e.g bitcoincash:qpmmlusvvrjj9ha2xdgv8xcrpfwsqn5rngt3k26ve2
+    """
+    if address.upper() != address and address.lower() != address:
+        return False
+    address = address.lower()
+    colon_count = address.count(':')
+    if colon_count == 0:
```

Comment:
```suggestion
    if (colon_count := (address := address.lower()).count(':')) == 0:
```

might be ugly this way, so doing it just for `colon_count` might suffice.